src/OpenTelemetry/Trace/ActivityExporterSync.cs

```diff
@@ -0,0 +1,75 @@
+﻿// <copyright file="ActivityExporterSync.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace OpenTelemetry.Trace
+{
+    /// <summary>
+    /// Enumeration used to define the result of an export operation.
+    /// </summary>
+    public enum ExportResultSync
```

Comment:
should we simply use a bool to indicate success/failure?

Reply:
I guess we want to keep close to the [spec](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/sdk.md#exportbatch).

Reply:
Now I wonder why spec didn't simply use a boolean? I guess its leftover from old behavior where ExportResult had more than 2 possible values!

Reply:
I guess it leaves the room so in the future folks can add more value.

Reply:
mm.. Is there any value in Export() returning anything at all? I mean, irrespective of the return value, the Processor simply moves on to next batch. Is it to let Processor do some logging that batch is being dropped? Or to allow future possibilities..

Reply:
I don't see BIG value, but I do see value for example the processor could log an internal error, or the processor can use this indication to report exporting statistics (e.g. metrics telling folks how many activities are exported successfully vs. dropped).

Reply:
> the processor could log an internal error, or the processor can use this indication to report exporting statistics

These are definitely interesting use cases, but seems to me to be more of a concern for the exporter itself since it has the underlying knowledge of either the error or statistics.

Reply:
I agree.

Reply:
ok. lets stick with spec and keep enum with 2 values for now :)